WalletWasabi.Documentation/ClientRelease/ReleaseNotesTemplate.md

```diff
@@ -0,0 +1,42 @@
+### _[Wasabi Wallet](https://WasabiWallet.io) is an easy to use, privacy-focused, open-source, non-custodial, Bitcoin wallet_
+
+# Download
+:window: [Windows](https://github.com/zkSNACKs/WalletWasabi/releases/download/v2.0.x/Wasabi-2.0.x.msi)
+:green_apple: [Apple M1/M2](https://github.com/zkSNACKs/WalletWasabi/releases/download/v2.0.x/Wasabi-2.0.x-arm64.dmg)
+:apple: [Apple Intel](https://github.com/zkSNACKs/WalletWasabi/releases/download/v2.0.x/Wasabi-2.0.x.dmg)
+:penguin: [Ubuntu / Debian](https://github.com/zkSNACKs/WalletWasabi/releases/download/v2.0.x/Wasabi-2.0.x.deb)
+:penguin: [Other Linux](https://github.com/zkSNACKs/WalletWasabi/releases/download/v2.0.x/Wasabi-2.0.x.tar.gz)
+
+---
+## Release Highlights
+
+## Release Summary
+
+Read the [related blog]() for more information.
```

Comment:
I would use the blog's link, so if there is no release summary or we forget to change this, it would still work.

```suggestion
Read the [related blog](https://blog.wasabiwallet.io/) for more information.
```